public/src/reducers/calls.jsx

```diff
@@ -1,37 +1,62 @@
 const defaultCalls = {
-  current_call: undefined,
-  next_call: undefined,
-  calls_made: 0
+  current_call: false,
+  calls_made: 0,
+  call_id: null,
+  user_id: null,
+  campaign_id: null,
+  call_sid: null,
+  contact_id: null,
+  status: undefined,
+  outcome: undefined,
+  call_ended: null,
+  call_started: null,
+  notes: null,
+  contact_name: undefined,
+  call_active: false
```

Comment:
Would suggest renaming this to current_call_active or something a little more explicity. Alternatively, we can also think about adding a new status for a call for when it has been completed but responses have not been submitted (to distinguish this from ATTEMPTED). What do you think?

Reply:
so rn call active with the status is our check for this.